### DISCUSSION TITLE
Info request: install paths

### DISCUSSION THREAD

**Discussion by Lewiscowles1986** [shivammathur/setup-php#431] [Q&A / Help] [answered]
If I use this to install phpunit:6.5.5 where would it install it?

**Comment by shivammathur** [comment, marked as answer]
ubuntu/macos: /usr/local/bin/phpunit
Windows: C:\tools\php\phpunit

**Comment by jakoch** [comment]
Please add the install paths to the tools section of your readme. Thank you. 😊

**Reply by Lewiscowles1986** [reply]
what if whatever the value were, it ran which phpunit, or told us where the bin directory was?
